test.js

```diff
@@ -7,3 +7,23 @@ test(function (t) {
   t.deepEqual(Snake({fooBar: 'baz', nested: {fooBar: 'baz'}}), {foo_bar: 'baz', nested: {foo_bar: 'baz'}})
   t.end()
 })
+
+test('It will not deeply convert when option is set', function (t) {
+  var sourceObject = {
```

Comment:
Keep it simple. Pass things in directly, rather than declaring with long names. e.g.

```js
t.deepEqual(
  Snake({fooBar: {barBaz: 'qux'}}, {deep: false})
  {foo_bar: {barBaz: 'qux'}}
)
```

I'd prefer to be concise for something simple like this, I don't feel like tests need read like a story at the expense of being 10x more lines.